README.md

```diff
@@ -235,3 +235,13 @@ Interaction with GNU `make` jobserver
 sccache provides support for a [GNU make jobserver](https://www.gnu.org/software/make/manual/html_node/Job-Slots.html). When the server is started from a process that provides a jobserver, sccache will use that jobserver and provide it to any processes it spawns. (If you are running sccache from a GNU make recipe, you will need to prefix the command with `+` to get this behavior.) If the sccache server is started without a jobserver present it will create its own with the number of slots equal to the number of available CPU cores.
 
 This is most useful when using sccache for Rust compilation, as rustc supports using a jobserver for parallel codegen, so this ensures that rustc will not overwhelm the system with codegen tasks. Cargo implements its own jobserver ([see the information on `NUM_JOBS` in the cargo documentation](https://doc.rust-lang.org/stable/cargo/reference/environment-variables.html#environment-variables-cargo-sets-for-build-scripts)) for rustc to use, so using sccache for Rust compilation in cargo via `RUSTC_WRAPPER` should do the right thing automatically.
+
+---
+
+Known Caveats
+-------------
+
+* `proc-macro` crates like `serde_derive` cannot be cached.
+* `bin` crates cannot be cached.
+* Incrementally compiled crates cannot be cached. First party crates are compiled incrementally when building in debug mode. You may find that release builds are faster than debug builds because of this.
+* Absolute paths to files must match to get a cache hit. This means that even if you are using a shared cache, everyone will have to build at the same absolute path (i.e. not in $HOME) in order to benefit each other.
```

Comment:
I think it'd be good to mention that this applies for both first-party crates and third-party crates.  So in most situations where one might want a shared cache, this would mean that you would have to relocate your cargo cache to someplace other than `$HOME/.cargo` to benefit.